tests/unit/unit-walker.cpp

```diff
@@ -115,6 +161,730 @@ TEST_P(UnitWalkerTests, FunSubstitution)
   EXPECT_EQ(fy, iw.visit(fx));
 }
 
+TEST_P(UnitWalkerTests, SimpleTree)
+{
+  // traversal of simple formula using TreeWalker, test that cache gets built up as expected
+
+  SolverConfiguration sc = GetParam();
+  if (sc.is_logging_solver) {
+    // test only valid for logging solver
+
+    Term x = s->make_symbol("x", bvsort);
+    Term xp1 = s->make_term(BVAdd, x, s->make_term(1, bvsort));
+
+    UnorderedTermPairMap UndMap;
```

Comment:
rename to `und_map` or any other name, but with lower-case-underscores-style. Same for all other occurrences in all other tests of this variable.